[CSVFeedApiModule] prepreq.headers is overwritten in case of basic authentication (#27058)
demisto/content: Packs/BitcoinAbuse/ReleaseNotes/1_0_37.md

#### Integrations

##### BitcoinAbuse Feed

> Fixed an issue where the headers are overwritten in case of basic authentication.

Reviewer: ```suggestion
Fixed an issue where the request headers were overwritten when using basic authentication.
```

Please change in all RN.
